internal/donor/donordata/provided.go

```diff
@@ -398,6 +398,17 @@ func (p *Provided) SigningDeadline() time.Time {
 	return p.SignedAt.AddDate(2, 0, 0)
 }
 
+// DonorSigningDeadline gives the date at which the LPA should be signed by the
+// donor once identity is confirmed, and whether it will be possible for them to
+// meet that deadline.
```

Comment:
The last part of the sentence doesn't seem to match the code. Were you previously checking if they would be 18 at some point in this func?

Reply:
indeed I was